### PR TITLE
Add landing pages for all verticals

### PR DESCRIPTION
**Pull request by juliawu**
Add the rest of the /explore/<vertical> pages to the new homepage.

More example links for each vertical, and the number of datapoints will be added in a followup PR.

Screenshot 2023-07-27 at 4 03 44 PM
Screenshot 2023-07-27 at 4 04 04 PM

Screenshot 2023-07-27 at 4 04 30 PM
https://github.com/datacommonsorg/website/assets/4034366/e149cff2-b568-4da7-847b-bc9bd73b5895">
Screenshot 2023-07-27 at 4 05 03 PM
Screenshot 2023-07-27 at 4 05 14 PM

### CODE REVIEW

**Review by rvguha — APPROVED** [review]

File: static/js/apps/explore/app.tsx [hunk 1]
```diff
@@ -158,14 +148,20 @@ export function App(): JSX.Element {
           >
             {formatNumber(currentTopic.meta.variableCount)}
           </span>{" "}
-          statistical variables. We collect our health information from sources
+          statistical variables. We collect our{" "}
+          {currentTopic.title.toLocaleLowerCase()} information from sources such
```
Comment (chejennifer) [inline static/js/apps/explore/app.tsx:152]: remove the such?

File: static/js/apps/explore/app.tsx [hunk 2]
```diff
@@ -158,14 +148,20 @@ export function App(): JSX.Element {
...
           such as:{" "}
           {currentTopic.meta.sources.map((s, i) => (
             <>
+              {currentTopic.meta.sources.length > 1 &&
+              i === currentTopic.meta.sources.length - 1
+                ? "and "
+                : ""}
               {s}
-              {i === currentTopic.meta.sources.length - 1 ? ", " : ""}
+              {i === currentTopic.meta.sources.length - 1 ? "" : ", "}
```
Comment (chejennifer) [inline static/js/apps/explore/app.tsx:161]: did you mean to add comma for the last item?

File: static/js/apps/explore/topics.json [hunk 3]
```diff
@@ -37,69 +55,142 @@
...
-        "variableCount": 3456,
-        "sources": ["United Nations"]
+        "dataPointCount": 0,
+        "variableCount": 2992,
+        "sources": [
+          "Climate Trace",
+          "Dynamic World Project",
+          "Federal Emergency Management Agency (FEMA)",
+          "U.S. Environmental Protection Agency (EPA)",
+          "Us.s National Aeornautics and Space Administration (NASA)",
```
Comment (chejennifer) [inline static/js/apps/explore/topics.json:124]: U.S. ?

File: static/js/apps/explore/topics.json [hunk 4]
```diff
@@ -37,69 +55,142 @@
...
-        "sources": ["United Nations"]
+        "dataPointCount": 0,
+        "variableCount": 12746,
+        "sources": [
+          "Climate Trace",
+          "U.S. Energy Information Administration (EIA)",
+          "U.S. National Reneweable Energy Laboratory (NREL)",
+          "Federal Emergency Management Agency (FEMA)",
+          "U.S. Environmental Protection Agency (EPA)",
+          "Us.s National Aeornautics and Space Administration (NASA)",
```
Comment (chejennifer) [inline static/js/apps/explore/topics.json:188]: U.S. ?

**Review by chejennifer — COMMENTED** [review]
nice!

File: static/js/apps/explore/app.tsx (hunk 2 above)
Comment (juliawu) [reply static/js/apps/explore/app.tsx:161]: If I have the logic right, it should only add commas if it's NOT the last item, right?

Comment (chejennifer) [reply static/js/apps/explore/app.tsx:161]: ah yes, sorry, I got original & changed mixed up

File: static/js/apps/explore/topics.json (hunk 4 above)
Comment (juliawu) [reply static/js/apps/explore/topics.json]: Fixed! Thank you!

File: static/js/apps/explore/topics.json (hunk 3 above)
Comment (juliawu) [reply static/js/apps/explore/topics.json]: Fixed. Thank you!

File: static/js/apps/explore/app.tsx (hunk 1 above)
Comment (juliawu) [reply static/js/apps/explore/app.tsx:152]: Fixed, thank you!

File: static/js/apps/explore/topics.json [hunk 5]
```diff
@@ -10,16 +10,34 @@
...
-        "variableCount": 3456,
-        "sources": ["United Nations"]
+        "dataPointCount": 0,
+        "variableCount": 67486,
+        "sources": [
+          "U.S. Bureau of Economic Analysis",
+          "U.S. Bureau of Labor Statistics",
+          "U.S. Census Bureau",
+          "U.S. Department of Labor",
+          "World Bank"
```
Comment (pradh) [inline static/js/apps/explore/topics.json:39]: I think we have other sources for economy too?

Eurostat, RBI (India), etc.  Ditto for other sources.

Generally, given these are broad verticals, I feel we might be able to find a way to fit most of our sources listed here -- https://autopush.datacommons.org/browser/Source ?

@n-h-diaz for further consultation

Comment (juliawu) [reply static/js/apps/explore/topics.json:39]: Do we want to list them all? The examples on the pages are very US centric, which is why I filtered to those sources.

Comment (pradh) [reply static/js/apps/explore/topics.json:39]: @rvguha wanted to list them all, double check with him?

**Pull request merged**